.circleci/local_publish_helpers.sh

```diff
@@ -12,6 +12,27 @@ function startLocalRegistry {
     grep -q 'http address' <(tail -f $tmp_registry_log)
 }
 
+function generatePkgCli {
```

Comment:
is this meant to be part of this PR? I don't see it used anywhere else

Reply:
hm, it should be used in package.json. Must have been reset at some point by mistake. Re-adding now.